.github/workflows/build-ci-container.yml

```diff
@@ -33,8 +38,8 @@ jobs:
       - name: Write Variables
         id: vars
         run: |
-          tag=`date +%s`
-          container_name="ghcr.io/$GITHUB_REPOSITORY_OWNER/ci-ubuntu-22.04"
+          tag="${{ github.run_id}}.${{ github.run_attempt }}"
```

Comment:
Can we use something better than this? Maybe the short commit SHA suffixed with the arch?

Reply:
We could use the short commit SHA.

I'm not sure about the arch suffix.  For Fedora containers we have the arch suffix:
`registry.fedoraproject.org/fedora:41-aarch64`

But for ubuntu there is no suffix and the arch is in the namespace: `docker.io/arm64v8/ubuntu:22.04`

I really  have no preference, I just want to be consistent with whatever the convention is.

Reply:
Ah, okay.

I think the short commit SHA would be better here given it makes things more easily searchable. Keeping the arch in the namespace sounds reasonable enough to me.